### DISCUSSION TITLE
[Feature Request] Prover FRI ?

### DISCUSSION THREAD

**Discussion by she110ff** [zkSync-Community-Hub/zksync-developers#171] [Feature requests]
### Network

zkSync Era

### Provide a brief description of the feature you would like to see implemented.

What is the difference between prover and prover_fri?

### Who does this feature request benefit the most (users, developers, protocol, etc.)?

developers

**Comment by porteradams12** [comment]
"Prover" is the old prover based on PLONK/Plookup.
"Prover_fri" is the new prover (Boojum).
See here: [https://github.com/matter-labs/zksync-era/blob/main/prover/README.md](https://github.com/matter-labs/zksync-era/blob/main/prover/README.md)
